modules/sfm/samples/data/images/dataset_files.txt

```diff
@@ -0,0 +1,4 @@
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2889.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2890.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2891.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2892.jpg
```

Comment:
in this case it's only a sample file to finally feed the reconstruction api with the images locations. The user must create/modify his own text file.